### DISCUSSION TITLE
arm64 version of the docker package?

### DISCUSSION THREAD

**Discussion by nacholibre** [PHP-CS-Fixer/PHP-CS-Fixer#8078] [Feature ideas]
Hi,

considering the increasing popularity of the ARM chips, more and more docker packages are offering arm64 builds. From what I can see the php-cs-fixer docker package is only amd64 and I'm getting the following warning when running via docker
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

I'm on a M1 Mac and Docker has Rosetta support, but in most cases performance is lost.

Would you consider creating arm docker images?

**Comment by Wirone** [comment]
Weird, I was pretty sure we have multiarch build 🤔. I will take a look at it, thanks for reporting.

**Reply by Wirone** [reply]
@nacholibre I've added `arm64` support in #8079, images for your architecture will be available starting from the next release 🙂.
